src/containers/Teams/TeamOverview/index.tsx

```diff
@@ -1,62 +1,44 @@
-import React, {FC, Fragment, ReactNode, useCallback} from 'react';
+import React, {FC, ReactNode, useCallback} from 'react';
 import {Icon, IconType} from '@thenewboston/ui';
 import {A, DocList} from 'components';
-import {TeamName} from 'types/teams';
-import {getTeamData} from 'utils/data';
+import {CoreTeam} from 'types/teams';
 
 import './TeamOverview.scss';
 
 interface ComponentProps {
-  teamFilter: TeamName;
+  team: CoreTeam;
 }
 
-const TeamOverview: FC<ComponentProps> = ({teamFilter}) => {
-  const {description, platforms, responsibilities} = getTeamData(teamFilter);
-
+const TeamOverview: FC<ComponentProps> = ({team}) => {
   const renderTeamDescription = useCallback((): ReactNode => {
     return (
       <>
         <h4 className="TeamOverview__sub-heading"> About the team </h4>
-        <p className="TeamOverview__description">{description}</p>
-        {platforms.map(({name, label, link}) => (
-          <div className="TeamOverview__social" key={`${name}-${label}`}>
-            <Icon
-              className="TeamOverview__social-icon"
-              icon={name === 'github' ? IconType.github : IconType.discord}
-              size={18}
-            />
-            {name === 'github' ? (
-              <A className="TeamOverview__social-title" href={link}>
-                {label}
-              </A>
-            ) : (
-              <p className="TeamOverview__social-title TeamOverview__social-title--sail-gray"> #{label} </p>
-            )}
-          </div>
-        ))}
+        <p className="TeamOverview__description">{team.about}</p>
+        <div className="TeamOverview__social">
+          <Icon className="TeamOverview__social-icon" icon={IconType.github} size={18} />
+          <A className="TeamOverview__social-title" href={team.github}>
+            {team.github}
+          </A>
+        </div>
+        <div className="TeamOverview__social">
+          <Icon className="TeamOverview__social-icon" icon={IconType.discord} size={18} />
+          <p className="TeamOverview__social-title TeamOverview__social-title--sail-gray"> #{team.slack} </p>
```

Comment:
@ThaDeveloper we do not have the `discord` field within each team right? can we replace the `slack` field to `discord` within Team model since we moved over to discord?

Reply:
and feel free to post the newly-created issue as a link in the private `website-api` channel for better visibility